Makefile

```diff
@@ -333,18 +341,24 @@ ifneq ($(RESTY_IMAGE_BASE),src)
 	RESTY_IMAGE_TAG=$(RESTY_IMAGE_TAG) \
 	KONG_VERSION=$(KONG_VERSION) \
 	KONG_PACKAGE_NAME=$(KONG_PACKAGE_NAME) \
-	KONG_TEST_CONTAINER_NAME=$(KONG_TEST_CONTAINER_NAME) \
+	KONG_TEST_IMAGE_NAME=$(KONG_TEST_IMAGE_NAME) \
 	test/build_container.sh
 endif
 
 setup-tests: cleanup-tests
 ifneq ($(RESTY_IMAGE_BASE),src)
-	./.ci/setup_kind.sh
+	KONG_TEST_IMAGE_NAME=$(KONG_TEST_IMAGE_NAME) \
+	KONG_TEST_CONTAINER_NAME=$(KONG_TEST_CONTAINER_NAME) \
+		docker-compose -f test/kong-tests-compose.yaml up -d
+	while ! curl localhost:8001; do \
+		echo "Waiting for Kong to be ready..."; \
+		sleep 5; \
+	done
```

Comment:
I just noticed this will allow tests to begin even if cassandra isn't healthy because Kong will respond ready before the datastores are necessarily 100% up / ready.

Could we do something along the lines of `docker-compose ps | grep -q healthy`